app/queries/keywords_query.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class KeywordsQuery
+  attr_reader :keywords
+
+  def initialize(user)
+    @keywords = user.keywords
+  end
+
+  def call
+    @keywords = order_by_name
```

Comment:
I guess, we only need to assign here, because we will not later access `keywords` from `KeywordsQuery.new(current_user).keywords`. What do you think?
```suggestion
    order_by_name
```

Reply:
true :) ✅
Fixed!